README.md

```diff
@@ -43,3 +43,20 @@ gds govuk connect --help
 It can be shortened to `gds govuk c`.
 
 Whichever installation method you choose, you can use the above method or the standalone `govuk-connect` binary.
+
+## Releases
+
+When you've merged the PR that bumps the version, publish a release tag and the gem to RubyGems from the `master` branch:
+
+```bash
+gem build .
+gem publish govuk-connect-<version>.gem # Credentials are in govuk-secrets/pass/2ndline
+git tag <version> # eg 0.0.3
+git push origin <version>
+```
+
+To raise a PR to bump the Homebrew formula version, run:
+
+`brew bump-formula-pr govuk-connect --url=https://rubygems.org/downloads/govuk-connect-<version>.gem`
```

Comment:
Does this need credentials too?

Reply:
No, although now you mention it we should pass `--no-fork` as we should all have write access to the repo and don't have to fork it to our personal accounts.

Reply:
I just gave the `GOV.UK Production` team write access to `alphagov/homebrew-gds`, so we can raise and merge our own PRs. So `--no-fork` will work.

Reply:
Fixed.